test/static_a_token.test.ts

```diff
@@ -81,28 +83,30 @@ describe("static_a_token", function () {
   });
 
   it("allows bridge to mint", async () => {
+    const mint_amount = BigNumber.from("100").mul(WAD).toString();
     await bridge.invoke(token, "mint", {
       recipient: BigInt(user1.starknetContract.address),
       amount: {
         high: 0n,
-        low: BigInt(decimalToWad(100)),
+        low: BigInt(mint_amount),
```

Comment:
why not using `decimalToWad` anymore?

Reply:
Because I'm using the extended BigNumber